examples/selenium-container/src/test/java/SeleniumContainerTest.java

```diff
@@ -14,20 +21,30 @@
 /**
  * Simple example of plain Selenium usage.
  */
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = DemoApplication.class, webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
```

Comment:
Please change from `DEFINED_PORT` to `RANDOM_PORT` and use `LocalServerPort`'s value when calling `exposeHostPorts`

Reply:
Updated